FakebookNotifications/FakebookNotifications.DataAccess/UserRepo.cs

```diff
@@ -0,0 +1,50 @@
+﻿using FakebookNotifications.DataAccess.Models;
+using FakebookNotifications.DataAccess.Models.Interfaces;
+using Microsoft.Extensions.Options;
+using MongoDB.Driver;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FakebookNotifications.DataAccess
+{
+    class UserRepo : IUserRepo
```

Comment:
This class will want to be public.

Reply:
Thank you. I also made the interfaces public as well